packages/api/tests/task.test.js

```diff
@@ -56,7 +53,7 @@ describe('Task tests', () => {
   function postTransplantTaskRequest({ user_id, farm_id }, data, callback) {
     chai
       .request(server)
-      .post(`/task/transplant_task`)
+      .post('/task/transplant_task')
```

Comment:
@MwayaB This is an example of a change made automatically by eslint when the pre-commit hook runs it. There's a style rule to prefer single quotes where they will work.
(This change isn't all that important. Just pointing out the effect of eslint's autofixing. There are many more cases of that that I won't comment on.)